sample_network_tests/vane_system_tests/test_vane.py

```diff
@@ -594,3 +594,47 @@ def test_cmd_template(self, dut, tests_definitions):
         self.tops.parse_test_steps(self.test_cmd_template)
         self.tops.generate_report(dut["name"], self.output)
         assert self.tops.expected_output == self.tops.actual_output
+
+
+@pytest.mark.vane_system_tests
+class TestcaseSkipTests:
+    """Suite of testcases to validate PyTest skip"""
+
+    def test_testcase_skip(self, dut, tests_definitions):
+        """
+        TD: Verify correct behavior when PyTest skip occurs.
+        Args:
+            dut(dict): details related to a particular DUT
+            tests_definitions(dict): test suite and test case parameters.
+        """
+
+        tops = tests_tools.TestOps(tests_definitions, TEST_SUITE, dut)
+
+        """
+        TS: Set skip condition to True so test case is skipped
+        """
+        skip = True
+        self.output = None
+        tops.output_msg = "Not skipping test case."
+
+        try:
+            """
+            TS: Implement PyTest skip logic
+            """
+            if skip:
+                output_msg = f"Skipping test case on {tops.dut_name}"
```

Comment:
As discussed in the style guide comments, do we want to replace output_msg by tops.output_msg and pass tops.output_msg to pytest.skip()
```suggestion
                tops.output_msg = f"Skipping test case on {tops.dut_name}"
```